BetSystem/BusinnessLogic/SportEventService.cs

```diff
@@ -49,7 +54,7 @@ public List<SportEventDto> GetAllEvents()
         public SportEventDto GetById(int id)
         {
             var sportEvent = _dbContext.Events.FirstOrDefault(x => x.Id == id);
-            if(sportEvent == null) { return null; } 
+            if (sportEvent == null) { return null; }
```

Comment:
Return null its not best solution - maybe we need throw exception (maybe need to implement custom exception NotFoundException) and handle it in middleware?

Reply:
Added Exception